utils/posts.js

```diff
@@ -41,15 +43,68 @@ export function getPostMetadata(id) {
             ...matterResult.data,
         },
     };
-
+    
     const title = post.meta.title;
-    var seoTitle = title.toLowerCase().replace(/[^a-z0-9]+/g, ' ').trim().replace(/ /g, '-');
-
+    const seoTitle = title.toLowerCase().replace(/[^a-z0-9]+/g, ' ').trim().replace(/ /g, '-');
+    
     post.meta.seoTitle = seoTitle;
 
+    var tags = post.meta.tags;
+    if(tags) {
+        tags = tags.split(',').map(tag => tag.trim());
+	tags = tags.map(tag => ({
+            content: tag,
+            id: tag.replace(/ /g, '-').toLowerCase(),
+	}));
+    } else {
+        tags = null
+    }
+    post.tags = tags;
     return post;
 }
 
+// Retrieves a list of all unique tags
+export function getTags() {
+    const posts = getPosts();
+    var tags = [];
+    for(var i = 0; i < posts.length; i++) {
+        if(posts[i].tags) {
+            tags = tags.concat(posts[i].tags);
+        };
+    }
+    const uniqueSorted = Array.from(new Set(tags.map(tag => tag.id))).map(id => {
+        return tags.find(tag => tag.id === id);
+    }).sort((a, b) => {
+        const result = a.id > b.id ? 1 : -1;
+	return result;
+    });
+    return uniqueSorted;
```

Comment:
We could probably simplifiy this code to...

```
const tagsMap = {};
posts.forEach((post) => {
    post.tags.forEach((tag) => {
        tagsMap[tag] = true;
    });
});

const tags = _.sortBy(_.keys(tagsMap));
return tags;
```

https://lodash.com/docs/4.17.15#sortBy